fern/definition/requests.yaml

```diff
@@ -31,6 +31,12 @@ types:
       - SQLINJECTION
       - TEMPLATE
       - NOSQL
+      - CORS_HEADERINJECTION
```

Comment:
i would like to have nested vuln types or a union here so that we're not mixing api injection vulns with header injection ones. have you considered adding more structure here?